andela/ah-centauri-frontend
# #164829186 Users should be able to see their reading stats

#### What does this PR do?
- Add stats page for Users to be able to see their reading stats.
#### Description of Tasks to be completed?
- Add stats page for Users to be able to see their reading stats.
- Test the app.
#### How should this be manually tested?
- Clone the project: git clone https://github.com/andela/ah-centauri-frontend
- cd into the project folder
- checkout to branch `feature/164829186-see-reading-stats`
- Install dependencies: npm install
- Launch development server: npm start:dev
- Sign in
- Navigate to the navbar and hover around the avatar then click on Stats

#### What are the relevant Pivotal Tracker Stories?
[#164829186](https://www.pivotaltracker.com/story/show/164829186)

#### Screenshots(if appropriate)
### **Stats with no Article to view**
![image](https://user-images.githubusercontent.com/33542127/58315736-c45a1c00-7e1a-11e9-8114-ea32d4ad7ca0.png)
### **Stats with Articles to view**
![image](https://user-images.githubusercontent.com/33542127/58315745-c9b76680-7e1a-11e9-9a1e-524b997546d0.png)
